recipes/pycld2/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pycld2" %}
+{% set version = "0.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - compiler('cxx')
```

Oops, apparently needs to be templated, so should be:
```suggestion
    - {{ compiler('cxx') }}    
```